Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Management/Merchant.hs

```diff
@@ -1268,15 +1279,15 @@ postMerchantConfigFarePolicyUpsert merchantShortId opCity req = do
 
           id <- generateGUID
           let farePolicyId = finalFarePolicy.id
-          let fareProduct = DFareProduct.FareProduct {enabled = True, merchantId = merchantOpCity.merchantId, merchantOperatingCityId = merchantOpCity.id, ..}
+          let fareProduct = DFareProduct.FareProduct {enabled = eNabled, merchantId = merchantOpCity.merchantId, merchantOperatingCityId = merchantOpCity.id, ..}
```

Comment:
variable name is not looking good `eNabled` if it's conflicting with above we can name it as `enabled'`.

Reply:
Okay i am changing it to enabled'